common/types/custom_panels.ts

```diff
@@ -50,7 +50,7 @@ export interface SavedVisualizationType {
   selected_date_range: { start: string; end: string; text: string };
   timeField: string;
   application_id?: string;
-  user_configs: any;
+  userConfigs: any;
```

Comment:
I notice that this change seems to propagate through the code. Seems nitpicky but this is introducing mixed case where previously we used snake case in objects (though I guess timeField above was here first)